x-pack/plugins/infra/public/components/log_stream/log_stream.tsx

```diff
@@ -71,6 +71,9 @@ interface LogStreamContentProps {
   center?: LogEntryCursor;
   highlight?: string;
   columns?: LogColumnDefinition[];
+  hasColumnHeaders?: boolean;
```

Comment:
What do you think about using a stronger verb here such as

```suggestion
  showColumnHeaders?: boolean;
```

?

Reply:
I wasn't sure what was the nomenclature you guys use for this kind of thing so I went with `hasSomethingSomething` which seems to be what Eui does. I'll replace it with `show` instead :)

Reply:
true, it's a bit arbitrary :innocent: